python/feathub/table/table_descriptor.py

```diff
@@ -92,6 +93,15 @@ def get_feature(self, feature_name: str) -> Feature:
             f"Failed to find the feature '{feature_name}' in {self.to_json()}."
         )
 
+    def get_schema(self) -> Schema:
```

Comment:
The information provided by this method is a subset of the information provided by `get_output_features()`.

Would it be simpler to add a util methods in e.g. `feathub.common.utils` instead of adding an API on TableDescriptor?